turbo.json

```diff
@@ -0,0 +1,17 @@
+{
+  "$schema": "https://turbo.build/schema.json",
+  "tasks": {
+    "build": {
+      "dependsOn": ["^build"]
+    },
+    "build:app": {
+      "dependsOn": ["build"]
+    },
+    "lint": {
+      "dependsOn": ["build"]
+    },
+    "test": {
+      "dependsOn": ["build"]
```

Comment:
I actually like this—this conditionally builds for `pnpm test` but only if it needs to. Trying to find that balance using `pnpm run` has always been a struggle for me—either doing too much or too little work (and documenting it)